cmd/snap-confine/snap-confine.rst

```diff
@@ -129,7 +129,7 @@ FILES
 
 	Description of the mount profile.
 
-`/var/lib/snapd/seccomp/profiles/*`:
+`/var/lib/snapd/seccomp/bpf/*`:
```

Comment:
I do want something about `snap-seccomp` because while snap-confine may not care about the input files, people developing and auditing policy definitely do and they need to know that they have to compile policy for snap-confine to pick it up.

As for the separate directory, this is unfortunate and I was hoping the bpf would obviate the need for a separate directory for input files; and it does, but I see now only for rollbacks to versions of snapd with bpf.

As discussed on IRC, let's rename things again (sorry) so the are comfortable for users familiar with the existing system and somewhat discoverable. Ie:

```
snap-seccomp /var/lib/snapd/seccomp/profiles.bpf/snap.foo.bar /var/lib/snapd/seccomp/profiles.bpf/snap.foo.bar.bpf
```

In other words, rename the dir to `/var/lib/snapd/seccomp/profiles.bpf/`, don't use the `.in` extension for input files, but do use `.bpf` extension for output.